src/ApprenticeAan/SFA.DAS.ApprenticeAan.Application.UnitTests/ApprenticeAccount/Queries/GetApprenticeAccount/GetApprenticeAccountQueryHandlerTests.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System.Net;
+using AutoFixture.NUnit3;
+using FluentAssertions;
+using Moq;
+using SFA.DAS.ApprenticeAan.Application.ApprenticeAccount.Queries.GetApprenticeAccount;
+using SFA.DAS.SharedOuterApi.Configuration;
+using SFA.DAS.SharedOuterApi.InnerApi.Requests;
+using SFA.DAS.SharedOuterApi.Interfaces;
+using SFA.DAS.SharedOuterApi.Models;
+using SFA.DAS.Testing.AutoFixture;
+
+namespace SFA.DAS.ApprenticeAan.Application.UnitTests.ApprenticeAccount.Queries.GetApprenticeAccount;
+
+public class GetApprenticeAccountQueryHandlerTests
+{
+    [Test]
+    [MoqAutoData]
+    public async Task Handle_ApprenticeFound_ReturnsApprentice(
+        [Frozen] Mock<IApprenticeAccountsApiClient<ApprenticeAccountsApiConfiguration>> apiClientMock,
+        GetApprenticeAccountQueryHandler sut,
+        GetApprenticeAccountQuery request,
+        GetApprenticeAccountQueryResult expectedResult,
+        CancellationToken cancellationToken)
+    {
+
+        apiClientMock.Setup(c => c.GetWithResponseCode<GetApprenticeAccountQueryResult>(It.Is<GetApprenticeRequest>(r => r.Id == request.ApprenticeId))).ReturnsAsync(new ApiResponse<GetApprenticeAccountQueryResult>(expectedResult, HttpStatusCode.OK, null));
+
+        var actualResult = await sut.Handle(request, cancellationToken);
+
+        Assert.That(expectedResult, Is.EqualTo(actualResult));
```

Comment:
Cancellelation Token verification like [this](url)?

Reply:
Handler gets cancellation token in the argument, so don't see a need to verify that.